### ISSUE TITLE
Automatically set dependencies for Phabricator

### ISSUE THREAD

**Issue by ghost** [ralphbean/bugwarrior#509]
Taskwarrior supports [dependencies](https://taskwarrior.org/docs/best-practices.html), on the command line using the `depends:${TASK_ID}` flag. It would be nice if this would be automatically set for tasks synced from Phabricator.

This allows to later filter tasks based on their [virtual tags](https://taskwarrior.org/docs/tags.html), e.g. "blocked" (another task needs to be done first), "unblocked" (i.e. not blocked by another task) or "blocking" (i.e. another task is blocked by this).

**Comment by ralphbean** [comment]
Closing as @ghost is now a deleted user.
